test/metabase/api/dashboard_test.clj

```diff
@@ -2734,8 +2733,8 @@
       (testing "Should require a non-empty query"
         (doseq [query [nil
                        ""
-                       "   "
-                       "\n"]]
+                       #_"   "
+                       #_"\n"]]
```

Comment:
?

Reply:
oh I forgot to fix those tests. Fixed it here https://github.com/metabase/metabase/pull/32070/commits/00fc2ce55bb3f10c6643ee0e02a2992b05eff532#diff-fea57ccbb0743d3d2b610d8e43dee94131521bae2989f3b89a890756c9baffb2R2734-R2738